lms/djangoapps/courseware/views.py

```diff
@@ -1577,12 +1594,11 @@ def financial_assistance_form(request):
     enrolled_courses = [
         {'name': enrollment.course_overview.display_name, 'value': unicode(enrollment.course_id)}
         for enrollment in CourseEnrollment.enrollments_for_user(user).order_by('-created')
-        if CourseMode.objects.filter(
+        if enrollment.mode != CourseMode.VERIFIED and CourseMode.objects.filter(
```

Comment:
NIT:  rather than having a full query for an if statement this would be more readable if it were pulled out into a variable 

``` python
is_non_verified_enrollment = enrollment.mode != CourseMode.VERIFIED
course_has_unexpired_verified_course_mode = CourseMode.objects.filter(...)

if is_non_verified_enrollment and course_has_unexpired_verified_course_mode:
   ...
```